app/LinkLiteManager/Data/Entities/ActivitySource.cs

```diff
@@ -0,0 +1,13 @@
+namespace LinkLiteManager.Data.Entities;
+
+public class ActivitySource
+{
+  public int Id { get; set; }
+
+  public string Host { get; set; }
```

Comment:
Can you fix the warnings about nullable properties on both entities.

For info, the compiler is complaining that you haven't explicitly said these can be `null` (e.g. `string?`) but you also haven't initialised them with default values, which means they will start as `null`.

You can read the EF advice on entity properties specifically [here](https://docs.microsoft.com/en-us/ef/core/miscellaneous/nullable-reference-types)

but the gist of it is as follows:

- If a property can be null, use the nullable type (e.g. `string?`)
- For most non nullable primitives, it won't complain as they have defaults (e.g. `int` defaults to `0`)
- For non nullable `string`, initialise to `string.Empty`
- For referencing other entities as non nullable, initialise to `null!` (because EF guarantees they won't be null in situations they re requested.
- For collections, initialise to `new()`